submissions/ostrich.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "OSTRICH",
+    "contributors": [
+        "Matthew Hague",
+        "Denghang Hu",
+        "Anthony W. Lin",
+        "Oliver Markgraf",
+        "Philipp Rümmer",
+        "Zhilin Wu"
+    ],
+    "contacts": [
+        "Oliver Markgraf <markgraf@cs.uni-kl.de>",
+        "Philipp Rümmer <philipp.ruemmer@it.uu.se>"
+    ],
+    "archive": {
+        "url": "https://philipp.ruemmer.org/ostrich-2024.tar.gz",
+        "h": { "sha256": "bd9abdeb26c1b4541fee5ad61ef190c6055ef00b306d62739ad5704cb6e0ca4c" }
+    },
+    "website": "https://github.com/uuverifiers/ostrich",
+    "system_description": "https://philipp.ruemmer.org/ostrich-2024.pdf",
+    "command": ["./ostrich -portfolio=strings +quiet"],
```

Comment:
@pruemmer  Thanks for submitting OSTRICH to SMT-COMP! The command list is expected to contain one string for the executable and then one string per command line argument (similarly to how Python's `subprocess.run` is called). Could you change it in the pull request? In your case, it would be something like `"command": ["./ostrich", "-portfolio=strings", "+quiet"]`.

I fixed that locally and tried running the current submission on our competition infrastructure with some trivial benchmarks. Everything else seems to be working so far!

Reply:
Sure, thanks for the fix! I wasn't sure about the expected format here.